app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationSnackbarMessages.kt

```diff
@@ -0,0 +1,11 @@
+package com.wire.android.ui.home.conversations
+
+sealed class ConversationSnackbarMessages {
```

Comment:
minor thing but there is not need to mention that the error is for SnackBar I think

Reply:
I'm not sure I understand what you mean with this

Reply:
what I mean is, we do not need to mention that it is supposed to be confused but the Snackbar, you could remove the Snackbar prefix, but sincie it is a minor thingy and deadlnie is approach I am fine leaving it the way it is ;p